src/api/v1beta1/properties_test.go

```diff
@@ -417,3 +420,85 @@ func TestGetOneAgentEnvironment(t *testing.T) {
 		assert.Len(t, env, 0)
 	})
 }
+
+func TestDynaKube_ShallUpdateActiveGateConnectionInfo(t *testing.T) {
+	dk := DynaKube{
+		Status: DynaKubeStatus{
+			DynatraceApi: DynatraceApiStatus{
+				LastTokenScopeRequest:               metav1.Time{},
+				LastOneAgentConnectionInfoRequest:   metav1.Time{},
+				LastActiveGateConnectionInfoRequest: metav1.Time{},
+			},
+		},
+	}
+
+	timeProvider := timeprovider.New()
+	tests := map[string]struct {
+		lastRequestTimeDeltaMinutes int
+		shallUpdate                 bool
```

Comment:
```suggestion
		shouldUpdate                 bool
```

Reply:
No. "should" means, "you can if you like, just saying", shall means "do this or die".

Reply:
And in the end this is a test specification where it is appropriate to use specification language.

Reply:
But I can go with `expectUpdate`, that brings across the point as good.

Reply:
`expectUpdate` or `updateExpected` are great 👍